### ISSUE TITLE
Unable to install Ruby 1.9.3

### ISSUE THREAD

**Issue by gongled** [essentialkaos/rbinstall#61]
### Bug report

**System info:**

* **Version used (`rbinstall --version`):** 0.17.0
* **OS (e.g. from `/etc/*-release`):** CentOS Linux release 7.4.1708 (Core)
* **Kernel (`uname -a`):** `Linux b7c93bac75ef 4.9.49-moby #1 SMP Wed Sep 27 23:17:17 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux`

**Steps to reproduce:**

Run the following command to install Ruby 1.9.3-p551:
```
rbinstall -R 1.9.3-p551
```

**Expected behavior:**

Ruby, RubyGems and Bundler are installed without any errors.

**Actual behavior:**

```
✔ Checking sha1 checksum (< 1 second)
✔ Unpacking 7z archive (2 seconds)
✔ Checking binary (< 1 second)
✖ Updating RubyGems to 2.6.14 (< 1 second)
Can't update rubygems. Update command output saved as /tmp/rbinstall-fail.log
✖ Installing bundler (< 1 second)

Can't install gem bundler. Gem command output saved as /tmp/rbinstall-fail.log.
```

**Additional info:**

```
$ cat /tmp/rbinstall-fail.log
ERROR:  While executing gem ... (OptionParser::InvalidOption)
    invalid option: --no-document
```

**Comment by gongled** [comment]
I also made some tests and prepared a list of problematic Ruby versions affected by this issue:
- 1.9.3-p105
- 1.9.3-p125
- 1.9.3-p194
- 1.9.3-p286
- 1.9.3-p327
- 1.9.3-p362
- 1.9.3-p374
- 1.9.3-p385
- 1.9.3-p392
- 1.9.3-p429
- 1.9.3-p448
- 1.9.3-p484
- 1.9.3-p545
- 1.9.3-p547
- 1.9.3-p551

**Comment by andyone** [comment]
Thanks for reporting on it! You are right. Old versions contains old version of RubyGems gem which doesn't support `--no-document` option. 

I will made fix ASAP.

**Comment by andyone** [comment]
Fixed in #62
